crates/driver/src/tests/setup/solver.rs

```diff
@@ -270,18 +270,25 @@ impl Solver {
                                 }).collect_vec(),
                             })
                         }));
-                        prices_json.insert(
-                            config
-                                .blockchain
-                                .get_token_wrapped(jit.quoted_order.order.sell_token),
-                            jit.execution.buy.to_string(),
-                        );
-                        prices_json.insert(
-                            config
-                                .blockchain
-                                .get_token_wrapped(jit.quoted_order.order.buy_token),
-                            (jit.execution.sell - jit.quoted_order.order.surplus_fee()).to_string(),
-                        );
+                        // Skipping the prices for JIT orders (non-surplus-capturing)
+                        if config
+                            .expected_surplus_capturing_jit_order_owners
+                            .contains(&jit.quoted_order.order.owner)
+                        {
+                            prices_json.insert(
```

Comment:
Probably makes sense to assert that `.insert()` returned `None` to ensure that we don't overwrite prices for existing tokens by accident. (also applies to the code handling regular orders).